integration-cli/docker_cli_events_test.go

```diff
@@ -779,7 +779,7 @@ func (s *DockerSuite) TestEventsTop(c *check.C) {
 
 // #13753
 func (s *DockerSuite) TestEventsDefaultEmpty(c *check.C) {
-	dockerCmd(c, "run", "-d", "busybox")
+	dockerCmd(c, "run", "-d", "busybox", "top")
```

Comment:
If we're checking to make sure that we don't get any events after the container is stopped, why not just remove the '-d' (and don't add the 'top').  Then 'dockerCmd' won't exit until the container is stopped and the 'docker events' should return nothing, as expected.